Update README.md

meganmadley opened the pull request:
Updating to remove anaconda and switch to conda/miniforge, not sure if we also need to edit the environment.yml to remove the 'defaults' channel?

rumachan reviewed (changes requested):
`conda install ..... jupyter`: replace jupyter by jupyterlab. This gives the more modern and capable interface. Help->Launch Classic Notebook then gives you the option of the old style interface.

`To reopen notebooks, open the conda prompt and run` can you check this works. I'm not sure what it means as I always do from the command line. If you say its good then I'm happy.

The anaconda to conda wording switch makes things sound a little odd. Maybe we could chat on Teams and you run through the steps, if it's not too much trouble?

meganmadley commented:
Hey @rumachan, I've updated the README again. The lines in the readme to install jupyter and python are redundant as they are in the environment.yml, so I've removed them (when I ran it myself, it said I didn't need to as was already installed). I've changed `jupyter` to `jupyterlab` in the environment.yml as in PR https://github.com/GeoNet/data-tutorials/pull/29 - sorry that it's a separate PR. Happy to chat in teams to run through steps.

rumachan reviewed (approved):
I know this is correct because I have to do it!
